icicle/include/icicle/fields/field.h

```diff
@@ -776,8 +422,7 @@ class Field
    * cases it's less than 1, so setting the [num_of_reductions](@ref num_of_reductions) variable for a field equal to 1
    * will cause only 1 reduction to be performed.
    */
-  template <unsigned MODULUS_MULTIPLE = 1>
-  static constexpr HOST_DEVICE_INLINE Field reduce(const Wide& xs)
+  static constexpr HOST_DEVICE_INLINE Field barret_reduce(const Wide& xs) // TODO = add reduce_mont_inplace
```

Comment:
you are under ifdef BARRET. why changed the name?

Reply:
a more clear function name:)